From 517ac0278347e1127bb7ba855c75cffee3f56204 Mon Sep 17 00:00:00 2001
From: Oliver Smith <ollieparanoid@postmarketos.org>
Date: Mon, 11 Jul 2022 08:05:11 +0200
Subject: [PATCH] increase timeout for PartitionJob: 10 min

Increase the PartitionJob's timeout from 2 min to 10 min, as there was
an report of hitting the timeout with the PinePhone Pro's 128 GiB eMMC.

Related: https://gitlab.com/postmarketOS/pmaports/-/merge_requests/3280#note_1021536268
---
 modules/mobile/PartitionJob.cpp | 2 +-
 modules/mobile/wait.qml         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/mobile/PartitionJob.cpp b/modules/mobile/PartitionJob.cpp
index 38ca1ae..db2e5bb 100644
--- a/modules/mobile/PartitionJob.cpp
+++ b/modules/mobile/PartitionJob.cpp
@@ -118,7 +118,7 @@ PartitionJob::exec()
         const QString pathRoot = "/";
 
         ProcessResult res
-            = System::runCommand( System::RunLocation::RunInHost, args, pathRoot, stdInput, chrono::seconds( 120 ) );
+            = System::runCommand( System::RunLocation::RunInHost, args, pathRoot, stdInput, chrono::seconds( 600 ) );
         if ( res.getExitCode() )
         {
             return JobResult::error( "Command failed:<br><br>"
diff --git a/modules/mobile/wait.qml b/modules/mobile/wait.qml
index 9bf561d..0978764 100644
--- a/modules/mobile/wait.qml
+++ b/modules/mobile/wait.qml
@@ -38,7 +38,7 @@ Page
             anchors.topMargin: 150
             wrapMode: Text.WordWrap
             text: "Formatting and mounting target partition. This may" +
-                  " take up to two minutes, please be patient."
+                  " take up to ten minutes, please be patient."
             width: 500
         }
     }
-- 
2.30.2

